src/Lodge/Postcode/PostcodeServiceProvider.php

```diff
@@ -35,6 +35,8 @@ public function register()
         $this->app->singleton('postcode', function () {
             return new Postcode();

```

Comment:
You can pass the config here. This service provider only applies to Laravel, so it's a good place to inject the api key.
```php
return new Postcode(config('postcode-lookup::apikey'));
```